doc/cephfs/fs-volumes.rst

```diff
@@ -312,8 +312,44 @@ The output format is json and contains fields as follows.
 * created_at: time of creation of snapshot in the format "YYYY-MM-DD HH:MM:SS:ffffff"
 * data_pool: data pool the snapshot belongs to
 * has_pending_clones: "yes" if snapshot clone is in progress otherwise "no"
+* has_orphan_clones: "yes" if snapshot has orphan clones otherwise this field is not shown
+* pending_clones: list of in progress or pending clones and their target group if exist otherwise this field is not shown
 * size: snapshot size in bytes
 
+Sample output if snapshot clones are in progress or pending state::
+
+  $ ceph fs subvolume snapshot info cephfs subvol snap
+  {
+      "created_at": "2022-05-12 15:21:33.609021",
+      "data_pool": "cephfs.cephfs.data",
+      "has_pending_clones": "yes",
+      "pending_clones": [
+          {
+              "name": "clone_1",
+              "target_group": null
```

Comment:
@kotreshhr @vshankar Should we show `target_group` field in case of `nogroup` (null) ? or avoid to show `nogroup` (null) ?